### PR TITLE
Check for Image in config object

### PR DESCRIPTION
**Pull request by jhadvig**
There was a [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1441062) opened that Identifier value is not set in the console.
screenshot_from_2017-04-10_17-58-37

Was talking to @mfojtik and he said that we are not setting the `Image` attribute in the `ImageConfig` since 1.4
@stefwalter PTAL

### CODE REVIEW

File: views/image-body.html [hunk 1]
```diff
@@ -19,8 +19,8 @@
     <dd ng-if="!labels['build-date'] && !layers[0].v1Compatibility.created && image.dockerImageMetadata.Created" title="{{image.dockerImageMetadata.Created}}">{{image.dockerImageMetadata.Created | dateRelative}}</dd>
     <dt translate>Digest</dt>
     <dd><tt>{{ image.metadata.name }}</tt></dd>
-    <dt translate>Identifier</dt>
-    <dd><tt>{{ config.Image }}</tt></dd>
+    <dt ng-if="config.Image" translate>Identifier</dt>
```
Comment (jwforres) [inline views/image-body.html]: use ng-if-start and ng-if-end instead of duplicating the if check

**Comment by mfojtik** [comment]
@miminar @legionus PTAL please. (config.Image maps to the DockerConfig struct that we don't have for v2 images AFAIK).

File: views/image-body.html (hunk 1 above)
Comment (jhadvig) [reply views/image-body.html]: yeah, was thinking about it, but we use the same (dup.) logic on other places in this file so want sure which one to use. 
Will update

**Comment by jhadvig** [comment]
@jwforres @stefwalter updated

**Comment by jwforres** [comment]
change LGTM, not sure if @stefwalter had a merge queue or tests set up for this repo, guessing no?

**Comment by stefwalter** [comment]
Smoke tested this manually by editing ```tests/image.jsonp```. Does what it says on the tin.

So @jwforres is Origin including even less information about Docker images now? I guess that'll affect the registry console and display of images in OWC, correct? 

@jhadvig Could you point to the exact change in origin that this is a result of?

**Comment by jhadvig** [comment]
@stefwalter I guess @mfojtik will have more info about that

**Comment by mfojtik** [comment]
@stefwalter the manifest removal is here: https://github.com/openshift/origin/pull/11925

**Pull request merged**